build.gradle

```diff
@@ -1766,13 +1769,69 @@ project(':storage') {
   }
 }
 
+project(':tools:tools-api') {
```

Comment:
Should we change the javadoc part to include the tools.api package?

Reply:
I have added the tools.api package to javadoc  (see https://github.com/apache/kafka/pull/13454/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R1823)

Reply:
The doc-related test is shown in previous comment (https://github.com/apache/kafka/pull/13454#issuecomment-1495527523)